src/basic/devnum-util.c

```diff
@@ -57,6 +58,42 @@ int device_path_make_major_minor(mode_t mode, dev_t devnum, char **ret) {
         return 0;
 }
 
+int device_path_make_major_minor2(mode_t mode, dev_t devnum, char **ret) {
```

Comment:
"2"? that's chickening out while looking for a function name...

maybe `device_path_by_major_minor_sysfs()`? or so?

And maybe add a comment at the top, why this function exists, i.e. that it is supposed to work even without udev around, which device_path_make_major_minor() doesn't deliver.

Reply:
:)